src/entities/card/organisms/card-preview.tsx

```diff
@@ -14,8 +14,11 @@ import {
   iconDeckArrow,
   iconDeckCheck,
 } from '@box/ui';
+import { useEvent } from 'effector-react';
 
-type CardType = 'item' | 'details';
+import { navigationModel } from '../../navigation';
```

Comment:
И сиди гадай по этим относительным импортам - ты ресурс внутренний затронул или внешний))

```suggestion
import { navigationModel } from '@box/entities/navigation';
```

Reply:
entities?

Reply:
ну да, оно в entities...

Reply:
а оно точно полезно...

Reply:
> И сиди гадай по этим относительным импортам - ты ресурс внутренний затронул или внешний))

Reply:
Другими словами - **для наглядности и предсказуемости**

Может ты тут трогаешь тут внутренний модуль, и особо бед нет в плане "связывания" бизнес логики и зон ответственности

А что если ты импортируешь ресурс из того же слоя? Например "фичу из фичи"?

И здесь кстати ты ведь это самое и делаешь - **сущность из сущности** , что само по себе не есть хорошо

Но такое трудно заметить, когда повсюду в коде относительные импорты

Reply:
Имхо:
- Относительные импорты - для доступа к внутренним относительно модуля ресурсам
- Абсолютные импорты - для доступа к общим ресурсам, на уровне слоев (например к shared/**, к entities/** и т.п. - чтобы сразу было видно что обращаешься не к внутреннему ресурсу)

Reply:
Ну и это можешь почитать если оч интересно)

https://github.com/feature-sliced/documentation/discussions/52